.github/workflows/test.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/bash -x
 
 maker=$1
+compiler=$2
 
 mydir=$(dirname $0)
 source ${mydir}/setup_env.sh
 
-print "======================================== Tests"
-
 # Instead of exiting on the first failed test (bash -e),
 # run all the tests and accumulate failures into $err.
 err=0
 
-cd test
 export OMP_NUM_THREADS=8
-./run_tests.py --xml ${top}/report-${maker}.xml
+
+print "======================================== Tests"
+cd test
+
+./run_tests.py
```

Comment:
Why do we get rid of the xml file?

Reply:
It was used by Jenkins. At least right now, it isn't used by GitHub.